lightning/src/util/persist.rs

```diff
@@ -0,0 +1,78 @@
+// This file is licensed under the Apache License, Version 2.0 <LICENSE-APACHE
+// or http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your option.
+// You may not use this file except in accordance with one or both of these
+// licenses.
+
+//! Expose a simple key-value store trait KVStorePersister that allows one to 
+//! implement the persistence for ChannelManager, NetworkGraph, and 
```

Comment:
nit: here and a few other places in this file you have whitespace at EOL. `%s/ $//g` should fix it :)